jaxrs-spec/src/main/asciidoc/boilerplate.adoc

```diff
@@ -0,0 +1,45 @@
+﻿//
+//    Copyright (c) 2019 Markus KARG
+//
+//    This program and the accompanying materials are made available under the
+//    terms of the Eclipse Public License v. 2.0, which is available at
+//    http://www.eclipse.org/legal/epl-2.0.
+//
+//    This Source Code may also be made available under the following Secondary
+//    Licenses when the conditions for such availability set forth in the
+//    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+//    version 2 with the GNU Classpath Exception, which is available at
+//    https://www.gnu.org/software/classpath/license.html.
+//
+//    SPDX-License-Identifier: EFSL-1.0
+//
+
+== Boilerplate
+
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
```

Comment:
This specification *MAY NOT* refer to the JCP specification to determine compliance.
Doing so would require an implementor to accept the JCP specification license for
the JCP specification instead of just being bound by the Eclipse Foundation Specification License.

Reply:
I understand the legal issue, but the Jakarta REST specification cannot be implemented without respecting the normative text found in the JCP's specification. Or in other words, it makes no sense to request compliance with Jakarta REST if we need to remove that line without having adopted the original text in our own non-Boilerplate specification.

Reply:
BTW, just asking someone to follow the content of a document does *not* legally bind him to sign a contract with the JCP.

Reply:
You need to take this up with Mike, offline.

We understand that all of the specs are incomplete without the corresponding spec
documents.  The compromise we've agreed to for Jakarta EE 8 is to define the specs
only in terms of the scope statement and the javadocs.  We've also agreed that the
Jakarta EE specifications can not and will not refer to the JCP specifications.

Until you convince Mike to change the plan, you need to change this specification
accordingly so that we can get Jakarta EE 8 done on time.

Thanks.

Reply:
This conflicts in two points with the EF rules. First, it is not Mike to decide about what the Jakarta RESTful Web Services Specification contains, but solely us. Second, all discussion and communication must be public.

The specification makes no sense at all without either that line or the content, so it would be correct to not publish it at all, independent of any time frame.

Reply:
There's no requirement that all discussion must be public.  I have private conversations with Mike all the time.  You can too.

The decision about what to do for specs for Jakarta EE 8 was Mike's decision, as Eclipse Foundation executive director.  Please talk to him.  You can Cc as many other people as you want to keep the conversation public, but please address Mike directly.

Reply:
Agreed.

BTW, keeping all public was not my idea, but I was asked for by Wayne.

Reply:
Any updates on this @mkarg? In the interested of time, perhaps it is better to say less at this point and get back to this later.

Reply:
@spericas Depends on what you strive for. If you want, I can simply make that section empty, but there is no legal force other than "Mike wants it".

Reply:
@mkarg I'd rather focus our energy on future versions of the spec, and get this one out of the door. Perhaps dropping the last sentence as suggested by @bshannon is the best plan now.

Reply:
While I personally think an empty specification document without any reference to normative text is nonsense, I do not want to stand in the way. OTOH I do not want to actively support nonense. So it might be best if someone else provides the proposed changes and files a fast-track PR. I will vote +-0 for that.